.github/gh_requirements.txt

```diff
@@ -2,3 +2,4 @@ invoke
 tqdm
```

Comment:
Ok with having a single gh_req even if we don't need coverage all the times.
Unfortunately there is no easy way to compose gh_reqs, or can you do `pip install -r ... -r gh_req.coverage.txt`?